nexus/db-queries/src/db/queries/external_ip.rs

```diff
@@ -1075,7 +1066,7 @@ mod tests {
                     &context.opctx,
                     id,
                     instance_id,
-                    context.default_pool_id().await,
+                    None,
```

Comment:
All of these work as expected when given `None` to make it keep choosing the default pool, but there should probably now be a test for a non-`None` value.